tests/cases/buddy/stripmining-huge.mlir

```diff
@@ -1,4 +1,15 @@
-// https://github.com/xlinsist/buddy-mlir/blob/39d0f3107014d41b4e00c56c88f3f8f078164fd7/examples/VectorExpDialect/strip-mining-large-number-rv32.mlir
+// performance:
+// total_cycles: 82830
+// vrf_write_records: 66917/662640 (10.099%)
+// adder_occupied: 176280/662640 (26.603%)
+// shifter_occupied: 0/662640 (0.000%)
+// multiplier_occupied: 0/662640 (0.000%)
+// divider_occupied: 0/662640 (0.000%)
+// bank_0_avg_pending_req: 14592/82831 (0.176)
+// bank_0_occupied: 14592/82831 (17.617%)
+// bank_1_avg_pending_req: 38192/82831 (0.461)
+// bank_1_occupied: 33107/82831 (39.969%)
+// chaining_size: 176391/662640 (26.619%)
```

Comment:
These can be removed, since we are improving our RTL, we should report it based in the CI for each commits.

Reply:
Thanks! Removing it.